Modele_MB.R

```diff
@@ -23,14 +23,23 @@ R_hat <- 1/(n-1)*t(Y-Un%*%mut_hat)%*%solve(Cn)%*%(Y-Un%*%mut_hat)
 
 # Construction de Cmn, entre nœuds internes et feuilles
 
+MRCAfull <- mrca(tree, full = TRUE)
```

Comment:
Replace buggy `getMRCA` with a full matrix of `mrca`.
(compare e.g. `getMRCA(tree, c(115, 11))` (wrong result `114`) to `mrca(tree, full = TRUE)[115, 11]` (true result `115`)